### PR TITLE
Score keeping

### PR DESCRIPTION
**Pull request by AvshRLev**

### CODE REVIEW

File: games/static/games/proto.js [hunk 1]
```diff
@@ -35,6 +58,10 @@ class Game {
   start() {
     let gameId = 0;
     let timerId = 0;
+    let scoreId = 0;
+    scoreId = setInterval(() => {
```
Comment (buzzdan) [inline games/static/games/proto.js:62]: why do we need this?
shouldnt the score get updated whenever we hit alien or something ?

File: games/static/games/proto.js [hunk 2]
```diff
@@ -335,6 +369,23 @@ class Board {
       }
     }
   }
+  calculateCurrentTotalAlienHealth() {
+    let alienTotal = []
+    this.squares
+      .filter((square) => square.hasAlien())
+      .map((square) => alienTotal.push(square.character.health))
```
[the remark this answers is not in the document]
Comment (buzzdan) [reply games/static/games/proto.js:376]: see the definition of map and what should it do

File: games/static/games/proto.js [hunk 3]
```diff
@@ -321,6 +353,8 @@ class Board {
   moveAliens() {
     this.determineDirection();
     this.alienLocations = this.calculateAlienLocations();
+    this.updateCurrentTotalAlienHealth()
```
Comment (buzzdan) [inline games/static/games/proto.js:356]: i'm not sure its the place for calculating the score
is the score effected only from the hit?
idea: when a rocket hits an alien update score

File: games/static/games/proto.js (hunk 1 above)
Comment (AvshRLev) [reply games/static/games/proto.js:62]: I tried that way but working throgh the rocket function where the alien gets hit kept giving different results for the same test (shoot all the aliens and see the score is 120). So i did it through the alien movement instead so then i got constant results but it was updating only when the aliens was moving so i did this for immediate score display

File: games/static/games/proto.js (hunk 3 above)
Comment (AvshRLev) [reply games/static/games/proto.js:356]: it does not work so well...

**Pull request merged**